modules/llm-cache/ds/kv_state_cache_manager.cc

```diff
@@ -76,13 +76,31 @@ KVStateCacheManager::KVStateCacheManager(int dimension, int cacheCapacity,
     }
   }
 
-  // syncThread = new std::thread(threadFunc);
   syncThread = new std::thread(SyncThreadFunc, this);
+  if (syncThread == nullptr) {
+    throw std::runtime_error("Create sync thread failed!");
+  }
 
   // TBD
   // use lease to prevent the deadlock if the client is down
 }
 
+Status KVStateCacheManager::MakeKVStateCacheManager(
+    KVStateCacheManager*& manager, int dimension, int cacheCapacity, int layer,
```

Comment:
`MakeKVStateCacheManager` -> `Make`, as when you use the static method you would type `KVStateCacheManager::Make()`

Reply:
Done